### ISSUE TITLE
Image on dockerhub

### ISSUE THREAD

**Issue by physikerwelt** [dginev/LaTeXML-Plugin-ltxmojo#14]
I saw that there is also a Dockerfile in this repo. I am using a similar docker file as a template for my own [docker-file](https://github.com/ag-gipp/docker-latexml/blob/master/Dockerfile). But I think it would be better to depend on the container associated with this repo.

**Comment by dginev** [comment]
Would you have any preferred changes for the repository? Unsure how to resolve the issue. Since @tkw1536 contributed the Dockerfile here, maybe he has some thoughts as well.

**Comment by physikerwelt** [comment]
No, I am happy with using the base image https://hub.docker.com/r/mathwebsearch/latexml-mws However, maybe one could add a few words about that in the readme of this repo. I think an example of a proper docker usage instructions are [these LaTeX containers](https://gitlab.com/docker-hub/texlive/blob/master/README.md), or [the Wikimedia Deutschland Wikibase containers](https://github.com/wmde/wikibase-docker). Maybe @tkw1536 has already a vision of documenting the use of docker containers in the LaTeXML ecosystem.
